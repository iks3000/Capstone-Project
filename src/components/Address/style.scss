@import '../../assets/styles/_variables.scss';
@import '../../assets/styles/_extends.scss';

.address {
    margin: 50px 0;

    &__title {
        @extend %title;
    }

    &__item {
        display: flex;
        align-items: center;
        margin-bottom: 25px;

        &:last-child {
            margin-bottom: 0;
        }

        .img-wrapper {
            width: 50px;
        }

        .content-title {
            @extend %content-title;
        }

        .content-link {
            font: 400 16px/19px $primary-fonts;
            color: $raven;
        }
    }
}

@media only screen and (max-width: 600px) {
    .address {
        .content-link {
            font-size: 12px;
        }
        .img-wrapper {
            width: 30px;
            img {
                width: 20px;
                height: 20px;
            }
        }
    }
}