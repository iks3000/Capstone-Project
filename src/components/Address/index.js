import './style.scss';
import PhoneIcon from '../../assets/images/icons/phone.svg';
import EmailIcon from '../../assets/images/icons/email.svg';
import TwitterIcon from '../../assets/images/icons/twitter.svg';
import FacebookIcon from '../../assets/images/icons/facebook.svg';
import SkypeIcon from '../../assets/images/icons/skype.svg';

const Address = () => {
    return (
        <div className="address" id="addressComponent">
            <h2 className="address__title">Contacts</h2>
            <div className="address__list">
                <div className="address__item">
                    <div className="img-wrapper">
                        <img src={PhoneIcon} alt="phone icon" />
                    </div>
                    <a className="content-title" href="tel:+1500-342-242">500 342 242</a>
                </div>
                <div className="address__item">
                    <div className="img-wrapper">
                        <img src={EmailIcon} alt="email icon" />
                    </div>
                    <a className="content-title" href="mailto:office@kamsolutions.pl">office@kamsolutions.pl</a>
                </div>
                <div className="address__item">
                    <div className="img-wrapper">
                        <img src={TwitterIcon} alt="twitter icon" />
                    </div>
                    <div>
                        <h3 className="content-title">Twitter</h3>
                        <a className="content-link" href="https://twitter.com/wordpress" target="_blank" rel="noreferrer" >https://twitter.com/wordpress</a>
                    </div>
                </div>
                <div className="address__item">
                    <div className="img-wrapper">
                        <img src={FacebookIcon} alt="facebook icon" />
                    </div>
                    <div>
                        <h3 className="content-title">Facebook</h3>
                        <a className="content-link" href="https://www.facebook.com/facebook" target="_blank" rel="noreferrer" >https://www.facebook.com/facebook</a>
                    </div>
                </div>
                <div className="address__item">
                    <div className="img-wrapper">
                        <img src={SkypeIcon} alt="skype icon" />
                    </div>
                    <div>
                        <h3 className="content-title">Skype</h3>
                        <a className="content-link" href="callto:kamsolutions.pl">kamsolutions.pl</a>
                    </div>
                </div>
            </div>
        </div>
    )
};

export default Address;