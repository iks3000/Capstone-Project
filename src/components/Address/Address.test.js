import React from "react";
import { shallow } from "enzyme";
import Address from "./index";

it("should render Address component", () => {
  expect(shallow(<Address />)).toMatchSnapshot();
});
