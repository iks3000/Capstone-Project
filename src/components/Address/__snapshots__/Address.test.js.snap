// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Address component 1`] = `
<div
  className="address"
  id="addressComponent"
>
  <h2
    className="address__title"
  >
    Contacts
  </h2>
  <div
    className="address__list"
  >
    <div
      className="address__item"
    >
      <div
        className="img-wrapper"
      >
        <img
          alt="phone icon"
          src="phone.svg"
        />
      </div>
      <a
        className="content-title"
        href="tel:+1500-342-242"
      >
        500 342 242
      </a>
    </div>
    <div
      className="address__item"
    >
      <div
        className="img-wrapper"
      >
        <img
          alt="email icon"
          src="email.svg"
        />
      </div>
      <a
        className="content-title"
        href="mailto:office@kamsolutions.pl"
      >
        office@kamsolutions.pl
      </a>
    </div>
    <div
      className="address__item"
    >
      <div
        className="img-wrapper"
      >
        <img
          alt="twitter icon"
          src="twitter.svg"
        />
      </div>
      <div>
        <h3
          className="content-title"
        >
          Twitter
        </h3>
        <a
          className="content-link"
          href="https://twitter.com/wordpress"
          rel="noreferrer"
          target="_blank"
        >
          https://twitter.com/wordpress
        </a>
      </div>
    </div>
    <div
      className="address__item"
    >
      <div
        className="img-wrapper"
      >
        <img
          alt="facebook icon"
          src="facebook.svg"
        />
      </div>
      <div>
        <h3
          className="content-title"
        >
          Facebook
        </h3>
        <a
          className="content-link"
          href="https://www.facebook.com/facebook"
          rel="noreferrer"
          target="_blank"
        >
          https://www.facebook.com/facebook
        </a>
      </div>
    </div>
    <div
      className="address__item"
    >
      <div
        className="img-wrapper"
      >
        <img
          alt="skype icon"
          src="skype.svg"
        />
      </div>
      <div>
        <h3
          className="content-title"
        >
          Skype
        </h3>
        <a
          className="content-link"
          href="callto:kamsolutions.pl"
        >
          kamsolutions.pl
        </a>
      </div>
    </div>
  </div>
</div>
`;
